docs/ingestion/elastic-search.md

```diff
@@ -0,0 +1,13 @@
+# Elastic Search
+
```

Comment:
Add a line something like `assuming your siglens instance is running on localhost with ingestPort 8081`